bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx105-en.html

```diff
@@ -0,0 +1,62 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+{% block page_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-picto') }}
+  {{ css_bundle('firefox_whatsnew_105_en') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block wnp_content %}
+<section class="wnp-content-main mzp-l-content mzp-t-content-lg">
+  <h2 class="wnp-main-title">How to talk to your kids about online safety</h2>
+
+  <img src="{{ static('img/firefox/whatsnew/whatsnew105/tech-talk.png') }}" width="720" height="405" alt="The Tech Talk" class="wnp-main-img">
```

Comment:
Suggestion (non-blocking): a responsive image would be a nice to have here, mainly so we're not serving people on low res monitors a @2x sized image?